bots/common/build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     compile "com.fasterxml.jackson.module:jackson-module-kotlin:$version_jackson"
     compile "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:$version_jackson"
 
+    compile group: 'org.glassfish.tyrus.bundles', name: 'tyrus-standalone-client', version: '1.8.3'
```

Comment:
Is this needed, or can it be a transitive dependency of the slack API library? 

Also, please could you move the version out into the list of dependency versions.